docs/disco.sh

```diff
@@ -0,0 +1,200 @@
+    # ##*/ gets the basename of "script"
+    gcloud compute copy-files $path/${script##*/} $boxname:~
+}
+
+get-external-ip () {
+    local boxname=$(hostname)
+    local desc="gcloud compute instances describe $boxname"
+    local ip=$($desc | grep natIP | cut -d ':' -f 2 | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')
+    echo $ip
+}
+
+configure () {
+    # TODO
+    # 1. GATK/MUTECT/NETMHC URLS
+    # 2. NFS mounts
+    local boxname=$(hostname)
+    local nfsserver=$boxname-nfs
+    local externalip=$(get-external-ip)
+    cat <<EOF > configuration.env
+## Template for Configuring Ketrew with Coclobas scripts
+
+## Set the name of the cluster you want to manage:
+export CLUSTER_NAME=$boxname-cluster
+export NFS_SERVER_NAME=$nfsserver
+
+export EXTERNAL_IP=$externalip
+
+## Set the gcloud zone
+export GCLOUD_ZONE=$GCLOUD_ZONE
+
+## Choose an authentication token for the Ketrew server:
+#
+export TOKEN=$(cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+
+## Number of compute nodes in the deployed cluster:
+#
+export CLUSTER_MAX_NODES=15
+
+## Description of the NFS services that we want the containers to mount
+##
+## A :-separated list of ,-separated 4-tuples
+##
+## Each tuple is:  <nfs-server-vm>,<nfs-remote-path>,<witnessfile>,<mount-point>
+##
+## - <nfs-server-vm> is the instance name
+## - <nfs-remote-path> is the path on the server that we want to mount
+## - <witness-file> is a relative path to a file that should exist on the mount (used for verification)
+## - <mount-point> is the path where we mount the NFS service (on all cluster nodes)
+
+export NFS_MOUNTS=$nfsserver,/nfs-pool/,.witness.txt,/nfs-pool
+
+## Variables used in the biokepi_machine.ml script:
+
+##  Biokepi configuration requires a few directories shared accross the cluster:
+## $BIOKEPI_WORK_DIR is mandatory;
+export BIOKEPI_WORK_DIR=/nfs-pool/biokepi/
+
+# Optional
+## INSTALL_TOOLS_PATH is optional (default is $BIOKEPI_WORK_DIR/toolkit):
+# export INSTALL_TOOLS_PATH=/nfs-constants/biokepi-software/
+## PYENSEMBL_CACHE_DIR is optional (default is $BIOKEPI_WORK_DIR/pyensembl-cache):
+# export PYENSEMBL_CACHE_DIR=/nfs-constants/biokepi-pyensemble-cache/
+## REFERENCE_GENOMES_PATH is optional (default is $BIOKEPI_WORK_DIR/reference-genome)
+# export REFERENCE_GENOMES_PATH=/nfs-constants/biokepi-ref-genomes/
+## ALLOW_DAEMONIZE is optional (default: false)
+## if true some nodes (such as downloads or moving data around) will run with
+## daemonize backend (i.e. on the server/docker container).
+# export ALLOW_DAEMONIZE=true
+
+## DOCKER_IMAGE is optional (default: hammerlab/biokepi-run):
+## The docker image to use for the Kubernetes jobs
+# export DOCKER_IMAGE=something/this:that
+
+## Usual Biokepi variables used to download Broad software:
+# export GATK_JAR_URL="http://example.com/GATK.jar"
+# export MUTECT_JAR_URL="http://example.com/Mutect.jar"
+EOF
+    gcloud config set compute/zone $GCLOUD_ZONE
```

Comment:
Not sure it's a good thing to globally set the zone for people's `gcloud` installations...